packages/terra-form-input/src/terra-dev-site/doc/example/uncontrolled/DefaultIncompleteExample.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Field from 'terra-form-field';
+import Input from 'terra-form-input';
+
+const DefaultIncompleteExample = () => (
+  <div>
+    <Field label="Default" htmlFor="uncontrolled-default-incomplete">
+      <Input
+        name="default incomplete input"
```

Comment:
I think `invalid` should be removed here. As-is, this is not the default uncontrolled incomplete input, correct? I think there should be examples that show - 
- incomplete,
- invalid,
- invalid taking precedence when both are supplied.

Reply:
Removed `isInvalid` from this example and added a new example with both `isInvalid` and `isIncomplete` set: https://github.com/cerner/terra-core/pull/2670/commits/f3184124d28bce916fe72b76f0f19fa0aea8c0fd